src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -99,14 +99,34 @@ const IOUSendModalStackNavigator = createModalStackNavigator([{
     Component: IOUCurrencySelection,
     name: 'IOU_Send_Currency',
 },
+{
+    Component: AddPersonalBankAccountPage,
+    name: 'IOU_Send_Add_Bank_Account',
+},
+{
+    Component: AddDebitCardPage,
+    name: 'IOU_Send_Add_Debit_Card',
+},
 {
     Component: EnablePaymentsPage,
-    name: 'IOU_Enable_Payments',
+    name: 'IOU_Send_Enable_Payments',
 }]);
 
 const IOUDetailsModalStackNavigator = createModalStackNavigator([{
     Component: IOUDetailsModal,
     name: 'IOU_Details_Root',
+},
+{
+    Component: AddPersonalBankAccountPage,
+    name: 'IOU_Details_Add_Bank_Account',
+},
+{
+    Component: AddDebitCardPage,
+    name: 'IOU_Details_Add_Debit_Card',
+},
+{
+    Component: EnablePaymentsPage,
+    name: 'IOU_Details_Enable_Payments',
```

Comment:
As discussed with @marcaaron , I think we're adding some complexity in the code duplicating the `AddDebitCardPage` and `AddPersonalBankAccountPage` in two stacks.

If we duplicate them here, we need to pass them as parameters in `IOUConfirmationList`, `SettlementButtton` and `IOUDetailsModal` instead of just using them in `KYCWall`. 

If we can simplify it using only two routes, that would help us in maintenance in the future.

What do you think @nickmurray47 ?

Also, this is NAB.

Reply:
I like the idea of using generic routes for these then they will exist outside of the underlying stack. 

But I think in that case we should not have both `<` and `x` in their headers because they would both do the same thing.

This wasn't really ironed out in the doc so I followed the existing convention (we already had `IOU_Enable_Payments` set up with a unique route).